### ISSUE TITLE
aria-controls value

### ISSUE THREAD

**Issue by EtienneR** [cristijora/vue-form-wizard#114]
Hi,
Thx for your work.

When I check my page with [Axe](https://www.deque.com/products/axe/) extension, he returns violations about `aria-control` value because the`aria-control` value should be the same value than `id` value.

I use the 0.7 and I see the same problem in the official demo :  https://jsfiddle.net/bt5dhqtf/97/
(`id="t-Personaldetails0" aria-controls="Personal details"`)

**Comment by cristijora** [comment]
Hi @EtienneR 
I am a bit confused by this. Following this example http://accessibility.athena-ict.com/aria/examples/tabpanel2.shtml aria-control value should be the same as the `id` value of the panel `aria-controls="panel1"` while aria-labelledby should have the `id` of the tab.

```html
<ul class="tablist" role="tablist">         
<li id="tab1" class="tab" aria-controls="panel1" role="tab" ...>Prices</li>         
<li id="tab2" class="tab" aria-controls="panel2" role="tab" ..>Features </li>         
</ul>      
   
<div id="panel1" class="panel" aria-labelledby="tab1" role="tabpanel" ..>       
<h3 ..>Prices</h3>                   
 List of prices      
</div> 
     
<div id="panel2" class="panel" aria-labelledby="tab2" role="tabpanel" ..>        
<h3 ..>Features</h3>      
List of product features....          
</div> 
```

I'll try to make it work like the example above if that's ok

**Comment by cristijora** [comment]
Released under [0.8.1](https://github.com/cristijora/vue-form-wizard/releases/tag/v0.8.1)
The fiddle above doesn't show the mentioned accessibility error anymore.
